pkg/deploy/controller/deploymentconfig/factory.go

```diff
@@ -40,7 +40,7 @@ func (factory *DeploymentConfigControllerFactory) Create() controller.RunnableCo
 		},
 	}
 	queue := cache.NewFIFO(cache.MetaNamespaceKeyFunc)
-	cache.NewReflector(deploymentConfigLW, &deployapi.DeploymentConfig{}, queue, 2*time.Minute).Run()
+	cache.NewReflector(deploymentConfigLW, &deployapi.DeploymentConfig{}, queue, 30*time.Second).Run()
```

Comment:
I still need @smarterclayton to approve this. It's the only issue with this PR. The upstream controller has the same sync interval so status for upstream Deployments will usually be updated faster than having it set to 2 minutes. See https://github.com/openshift/origin/pull/6233#issuecomment-219423116